pkg/daemon/ceph/client/pool.go

```diff
@@ -283,7 +284,23 @@ func setCommonPoolProperties(context *clusterd.Context, clusterInfo *ClusterInfo
 		}
 	}
 
-	// set max_bytes quota
+	// set maxBytesHuman quota
+	if pool.Quotas.MaxBytesHuman != nil {
+		// check for format errors
+		maxBytesQuota, err := resource.ParseQuantity(*pool.Quotas.MaxBytesHuman)
+		if err != nil {
+			if err == resource.ErrFormatWrong {
+				return errors.Wrapf(err, "maxBytesHuman quota incorrectly formatted for pool %q, valid units include Ki, Mi, Gi, Ti, Pi", poolName)
+			}
+			return errors.Wrapf(err, "failed setting quota for pool %q, maxBytesHuman quota parse error", poolName)
+		}
+		// set max_bytes quota, 0 value disables quota
+		err = setPoolQuota(context, clusterInfo, poolName, "max_bytes", strconv.FormatInt(maxBytesQuota.Value(), 10))
+		if err != nil {
+			return errors.Wrapf(err, "failed to set max_bytes quota for pool %q", poolName)
+		}
+	}
+	// set maxBytes quota
 	if pool.Quotas.MaxBytes != nil {
```

Comment:
Let's make sure only one of the values is read.
```suggestion
	else if pool.Quotas.MaxBytes != nil {
```

Reply:
change made